src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -239,6 +239,7 @@
                            ")")
          :item-selected? item-selected?
          :clearable? true
+         :placeholder (text :t.administration/no-form)
```

Comment:
but these are not interchangeable, are they?

Reply:
I'm just guessing here because this isn't done yet, but I think the default placeholder is `Select...`, and that tells the user that they must do a selection. But an empty value is just fine now. So I'm saying a `""` is more like a valid empty value, a variation of the `No form` etc.

Reply:
Oh right I think I see what you mean. The question is: should an empty dropdown look like this?

![dropdown-no-form](https://user-images.githubusercontent.com/207385/113698534-3ec57980-96dd-11eb-89d2-3a675c04b52f.png)

Or like this?

![dropdown-empty](https://user-images.githubusercontent.com/207385/113698579-45ec8780-96dd-11eb-85d1-12aaecc27539.png)

I chose the former. 

Next up: should a catalogue item with no form be shown in the table like this?

![table-no-form](https://user-images.githubusercontent.com/207385/113698955-a8de1e80-96dd-11eb-8832-c4043f8f3d55.png)

Or like this?

![table-empty](https://user-images.githubusercontent.com/207385/113698986-ae3b6900-96dd-11eb-9571-b92c3091b043.png)

Here again I chose the former.

There's one more place where the form is shown: the view catalogue item page. Here I chose an empty input box for consistency with the other fields:

![view-empty](https://user-images.githubusercontent.com/207385/113699215-f064aa80-96dd-11eb-82c5-4c1424f350a1.png)

... but I guess it could be argued that this should say "No form" as well!

Reply:
I would have chosen the simple path i.e. having all these as empty and no localization, that's why the original comment. Since it's not difficult to do and its there now, I think the first one can benefit from the "No form" but the other two I would still leave empty (i.e. follow the existing convention until we have a real case from e.g. accessibility rules to say otherwise).